src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -712,6 +723,8 @@ internal virtual void InitializeDebugTargetsCore(ILaunchSettings profiles)
                 {
                     _firstSnapshotCompleteSource.TrySetResult(true);
                 }
+
+                _debugTargetsCoreInitialized = true;
```

Comment:
It looks like this field has the same meaning as _firstSnapshotCompleteSource?

Reply:
Doesn't seem to be the case. It looks like _firstSnapshotCompleteSource is only used with `DebugPageViewModel` which is only used for tests. _firstSnapshotCompleteSource is always null otherwise.